components/digital_ocean/actions/list-all-droplets/list-all-droplets.mjs

```diff
@@ -0,0 +1,66 @@
+import digitalOceanApp from "../../digital_ocean.app.mjs";
+import digitalOceanOptions from "../../common/options.mjs";
+import doWrapperModule from "do-wrapper";
+
+export default {
+  key: "digital_ocean-list-all-droplets",
+  name: "List all Droplets",
+  description: "List all Droplets",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    digitalOceanApp: digitalOceanApp,
+    includeAll: {
+      label: "Include all",
+      type: "boolean",
+      description: "Select either fetch all items or paginated results.",
+      default: true,
+      reloadProps: true,
+    },
+    tagName: {
+      label: "Tag name",
+      type: "string",
+      description: "Optionally used to filter by a specific tag.\n\n**Example:** `production`",
+      optional: true,
+    },
+  },
+  async additionalProps() {
+    let dynamicProps = {};
+    if (!this.includeAll) {
+      dynamicProps = {
+        ...dynamicProps,
+        currentPage: {
+          label: "Current page",
+          type: "integer",
+          description: "Which 'page' of paginated results to return.",
+          default: digitalOceanOptions.defaultCurrentPage,
+        },
+        pageSize: {
+          label: "Page size",
+          type: "integer",
+          description: "Desired pagination size when pulling results",
+          default: digitalOceanOptions.defaultPageSize,
+        },
+      };
+    }
+    return dynamicProps;
+  },
+  async run({ $ }) {
+    const pageSize = this.pageSize || digitalOceanOptions.defaultCurrentPage;
+    const DigitalOcean = doWrapperModule.default;
+    const api = new DigitalOcean(this.digitalOceanApp.$auth.oauth_access_token, pageSize);
```

Comment:
You can modify `digitalOceanWrapper` function
```javascript
    digitalOceanWrapper(pageSize = digitalOceanOptions.defaultCurrentPage) {
      const DigitalOcean = doWrapperModule.default;
      const api = new DigitalOcean(this.$auth.oauth_access_token, pageSize);
      return api;
    },
```
and then here you can make this modification
```suggestion
    const api = this.digitalOceanApp.digitalOceanWrapper(this.pageSize);
```